string const_name
geometry_msgs/PoseStamped end_posestamped
geometry_msgs/PoseStamped entry_posestamped
